src/server/scripts/Commands/cs_misc.cpp

```diff
@@ -2166,9 +2167,9 @@ class misc_commandscript : public CommandScript
                 {
                     Unit* target = nullptr;
                     if (unit->GetTypeId() == TYPEID_PLAYER)
```

Comment:
if is not needed anymore

Reply:
Source? (a bit late for April Fools' day)

Reply:
good catch